www/include/configuration/configObject/connector/formConnector.php

```diff
@@ -171,13 +171,13 @@
         $cntObj = new CentreonConnector($pearDB);
         $tab = $form->getSubmitValues();
         $connectorValues = array();
-        $connectorValues['name'] = $tab['connector_name'];
-        $connectorValues['description'] = $tab['connector_description'];
+        $connectorValues['name'] = filter_var($tab['connector_name'], FILTER_SANITIZE_STRING);
+        $connectorValues['description'] = filter_var($tab['connector_description'], FILTER_SANITIZE_STRING);
+        $connectorValues['enabled'] = ($tab['connector_status']['connector_status'] === 0 ? 0 : 1);
+        $connectorValues['command_id'] = isset($tab['command_id']) ? (int)$tab['command_id'] : null;
```

Comment:
in case of 'command_id' cant be cast to int it will return 0, is this could be problematic or not ?

Reply:
$tab['command_id'] is an array